wrappers/kv_no_quorum.go

```diff
@@ -169,6 +169,9 @@ func (k *noKvdbQuorumWrapper) LockWithTimeout(
 }
 
 func (k *noKvdbQuorumWrapper) Unlock(kvp *kvdb.KVPair) error {
```

Comment:
In the ticket this strack frame shows a non-nil kvp:
```
github.com/portworx/porx/vendor/github.com/portworx/kvdb/wrappers.(*noKvdbQuorumWrapper).Unlock(0x21f27e6, 0xc0019835c0)
        /root/workspace/_release-build-jobs_hk-rel-3.1.0/go/src/github.com/portworx/porx/vendor/github.com/portworx/kvdb/wrappers/kv_no_quorum.go:172 +0x25 fp=0xc003be5bf0 sp=0xc003be5bd0 pc=0x2550725
```
So, this fix most likely would not have made a difference?

Reply:
I know the test pointer showed zero, but that does not necessary mean that it was not trigger.
I tested on the playground, if the variable was previously assigned, there's actually address shown.

https://go.dev/play/p/7A7L1wDvKlb

Reply:
I think it would be easier if the kvdb library itself adds the check for nil. The caller shouldn't worry about how kvp is handled by the kvdb lib. 

And yeah, that's what I think too.. I'll create some tickets to keep track of the kvp issue.

Reply:
Also, i haven't look through what's the correct behavior when using no quorum kvdb wrapper. E.g.: what happens when we return an error here because Unlock fails? 
Are we expect to fail all the create/attach/format calls when kvdb goes down?